.github/workflows/run_integration_tests.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         run: |
           bash tools/ci/install_byond.sh
           source $HOME/BYOND/byond/bin/byondsetup
-          tools/build/build --ci dm -DCIBUILDING -DANSICOLORS -Werror
```

Comment:
wait, we absolutely need this. we have regularly regressed warnings without it.

you need to come up with a better solution than this

Reply:
the vanilla windows build doesn't catch the same warnings because we regress between versions (alternate tests)

Reply:
Added ignorable warnings